recipes-bsp/tegra-binaries/gstreamer1.0-plugins-tegra-binaryonly_34.1.0.bb

```diff
@@ -8,9 +8,6 @@ DEPENDS = "\
 	${@bb.utils.contains('DISTRO_FEATURES', ['x11', 'alsa'], 'virtual/libx11 alsa-lib', '', d)} \
 	libdrm virtual/egl virtual/libgles2 \
 "
-# XXX-- temporary
-DEPENDS += "tegra-libraries-camera tegra-libraries-nvdsseimeta libv4l"
-# --XXX
 
 L4T_DEB_TRANSLATED_BPN = "nvidia-l4t-gstreamer"
 
```

Comment:
The additions to LICENSE and LIC_FILES_CHKSUM, and the `decompress_license` function along with the `do_unpack` flags changes below here can also be removed.